spec/board_spec.rb

```diff
@@ -11,11 +11,13 @@
       board = Board.new
       expect(board).is_a? Board
     end
+  end
 
+  describe '#generate_cells' do 
```

Comment:
Great to test this! We should replace any "placement_coordinates" instances with "coordinates" here

Reply:
yeahh those need to go